src/blocks/blocks/lottie/block.json

```diff
@@ -36,6 +36,19 @@
 		},
 		"width": {
 			"type": "number"
+		},
+		"widthUnit": {
+			"type": "string"
+		},
+		"ariaLabel": {
+			"type": "string",
+			"default": ""
+		},
+		"bgColor": {
```

Comment:
For the sake of consistency, you can use the attribute name as `backgroundGradient` and `backgroundColor`